scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/client/LoadOps.scala

```diff
@@ -119,6 +126,56 @@ final private[client] class LoadOps(client: Client, jobService: JobOps) {
       location = location
     )
 
+  /**
+   * Upload List of rows to Cloud Storage as Avro file and load to BigQuery table. Note that element
+   * type `T` must be annotated with [[BigQueryType]].
+   */
+  def uploadTypedRows[T <: HasAnnotation: TypeTag](
+    tableSpec: String,
+    rows: List[T],
+    tempLocation: String,
```

Comment:
Since this is temp, shouldn't we clean if afterward ?

Reply:
on the other hand if I do:
```
  avro(
        List(blobId.toGsUtilUri),
        tableSpec,
        schema = Some(bqt.schema),
        createDisposition = createDisposition,
        writeDisposition = writeDisposition
  )

  storage.delete(blobId)
```
I am not confident that `avro` is fully synchronous and BQ doesn't read that file on the background

Reply:
`execute` contains `jobService.waitForJobs(loadJob)` so I think this is fine.

I'm wondering also if we should create a `SaveOps` to allow saving some file formats (avro/json)